custom/icds_reports/utils/aggregation_helpers/monolith/awc_location.py

```diff
@@ -100,8 +102,7 @@ def aggregate(self, cursor):
             cursor.execute(rollup_query)
 
         self.assert_no_awc_missing_from_new_table(cursor)
-        old_def, new_def = self.generate_diff_of_tables(cursor)
-        # TODO figure what to do with these
+        self.generate_diff_of_tables(cursor)
```

Comment:
I think some reasonable argument could be made that we should exit early here if this returns nothing. not doing that for now though since we've always recreated the table